Dockerfile.template

```
@@ -171,7 +171,7 @@ RUN set -eux; \
 		| awk '/=>/ { so = $(NF-1); if (index(so, "/usr/local/") == 1) { next }; gsub("^/(usr/)?", "", so); print so }' \
```

I'd prefer we match https://github.com/docker-library/python/pull/858 instead: :eyes:

```suggestion
		| awk '/=>/ { so = $(NF-1); if (index(so, "/usr/local/") == 1) { next }; gsub("^/(usr/)?", "", so); printf "*%s\n", so }' \
```

---

That’s indeed smarter.
I’ve updated the PR with this suggestion.